target/linux/realtek/image/rtl838x.mk

```diff
@@ -75,22 +75,24 @@ define Device/d-link_dgs-1210-28mp-f
 endef
 TARGET_DEVICES += d-link_dgs-1210-28mp-f
 
-# The "IMG-" uImage name allows flashing the iniramfs from the vendor Web UI.
-# Avoided for sysupgrade, as the vendor FW would do an incomplete flash.
-define Device/engenius_ews2910p
+define Device/engenius_ews2910p-v1
+  $(Device/engenius_ews2910p)
   SOC := rtl8380
-  IMAGE_SIZE := 8192k
-  DEVICE_VENDOR := EnGenius
   DEVICE_MODEL := EWS2910P
-  DEVICE_PACKAGES += realtek-poe
+  DEVICE_VARIANT := v1
   UIMAGE_MAGIC := 0x03802910
-  KERNEL_INITRAMFS := \
-	kernel-bin | \
-	append-dtb | \
-	libdeflate-gzip | \
-	uImage gzip -n 'IMG-0.00.00-c0.0.00'
+  SUPPORTED_DEVICES += engenius_ews2910p
```

Comment:
this should be: `engenius,ews2910p`

Reply:
Indeed. Fixed in new version of this PR.

```
# sysupgrade -T /tmp/openwrt-realtek-rtl838x-engenius_ews2910p-v1-squashfs-sysupgrade.bin 
Mon May 22 10:35:52 UTC 2023 upgrade: Device engenius,ews2910p not supported by this image
Mon May 22 10:35:52 UTC 2023 upgrade: Supported devices: engenius,ews2910p-v1 engenius_ews2910p
Image check failed.
```